app/models/concerns/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :comment
+  has_many :like
```

Comment:
Great job @Zilola-Nazarova 🤝

- [ ] As stated in the project requirements, the associations between models are set up correctly according to the given. Kindly add the missing association between the user and the post.
![user-post](https://github.com/Zilola-Nazarova/blog-app/assets/63917725/a447685c-32c5-45d2-b09e-88a8d225f707)

Reply:
Thank you so much @1GORDON for this catch!
I tried to use `increment!` but it didn't work for `update_posts_counter` because I forgot to add this foreign key!
Now everything works as expected 😃